### DISCUSSION TITLE
Windows Terminal Preview v1.15.200

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#13691] [Announcements]
This is a servicing release for the Preview channel of Windows Terminal.

> **Note**
> People in the Beta channel of the Windows Insider program will receive _1.15_ as a _Stable_ channel update while we test out coming features for the next version of Windows. It is roughly equivalent to the build included here, but it does not include the experimental text rendering engine.

> **Warning**
> There is a known issue in this release of Windows Terminal that may result in difficulty restoring the Terminal window from a minimized state.

* Windows Terminal no longer depends on a sidecar package of MIDI instrument voices to play "In the Hall of the Mountain King" when an application requests it (that is: `DECPS` support now relies on DirectSound!) (#13471)
* Enabling "read-only" mode for a pane will no longer result in a read only dialog appearing every time you focus, un-focus, move, highlight, or otherwise interact with the terminal inside it. Sorry about that! (#13483)
* We will no longer reserve space for the scroll bar marks when there is no scroll bar (#13454)
* `doskey` aliases can now be bypassed by inserting a space before them, as has apparently been documented since 1651 (#13476)
* We've enabled word wrapping on more of the tooltips in the application (#13463)
* The "Quake :tm: mode" window will no longer launch in full screen when the full screen launch mode is selected (#13473)
* The "debug tap" now includes line breaks to make it easier to read (#13475)

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.15.2002.0'>Windows Terminal Preview v1.15.200</a>.</em>

**Comment by cniggeler** [comment]
Hi, I was the requestor for the doskey fix and I am having trouble with this preview release. I set a number of preferences (font size, color, etc.) and when attempting to restart this Terminal, I do not get a window - my mouse cursor continuously flashes from normal to busy, and the tray icon blinks continuously such that I can't select it or close the app from the tray icon! I can't find it in task manager and the only way out is to restart my machine.

Unfortunately, I don't know which preference caused this behavior, and since I can't start the app, I can't get to the settings. Is there a way to reset, or do I need to reinstall?

**Reply by lhecker** [reply]
You can find the `settings.json` by opening the following path in File Explorer:
```
%LOCALAPPDATA%\Packages\Microsoft.WindowsTerminalPreview_8wekyb3d8bbwe\LocalState
```

If you delete the `settings.json` there it should hopefully work again. It'd be great if you could make a backup of the 2-3 files in that directory so that we can figure out what causes your issue afterwards. 🙂

**Comment by cniggeler** [comment]
It took awhile because I have to reboot after every attempt. The culprit is setting "Run this profile as Administrator", EITHER from the Command Prompt tab OR the Defaults tab!

Taking out the "elevate" line will once again allow it to run. Curiously, once it's loaded, the Window title changes to "Administrator: Command Prompt" and it DOES seem to have Admin privileges (googled and found to try "net session" which runs) without that line in settings.json. My account is an admin account.

I attached the zip containing settings.json and state.json.
[term-bad-settings.zip](https://github.com/microsoft/terminal/files/9274947/term-bad-settings.zip)

**Reply by cniggeler** [reply]
> It took awhile because I have to reboot after every attempt. The culprit is setting "Run this profile as Administrator", EITHER from the Command Prompt tab OR the Defaults tab!

I was able to replicate this issue on a different system (Win11 vs. Win10).

UPDATE: This behavior is ALSO present in version 1.14.1962.0! Should I report this elsewhere? Can't now, need to reboot ;-)

**Reply by lhecker** [reply]
FYI I just noticed that the release has a small error. The 200**3** variant is for Win11 and not Win10.
But apart from that, I couldn't replicate your issue using the 2003 variant on Win11 given your term-bad-settings.zip.

The issue you described sounds like Terminal is crashing on you during launch. I'd suggest creating an issue here on GitHub and sending us a crash dump. The easiest way to do so is by [reporting a problem in the Feedback Hub](https://go.microsoft.com/fwlink/?linkid=2125419). Make sure to click the "Start recording" button, then reproduce the issue before submitting the feedback. Once it's submitted, paste a link to the feedback in the GitHub issue. Thanks! 🙂

![image](https://user-images.githubusercontent.com/18356694/140811502-a068f78b-89d2-4587-925a-73e19652b830.png)

![image](https://user-images.githubusercontent.com/18356694/140811557-cdc22a0f-fa6a-4f6a-953e-73b51f5548a3.png)

![image](https://user-images.githubusercontent.com/18221333/62478649-6de55400-b760-11e9-806e-5aab7e085a9f.png)

**Reply by cniggeler** [reply]
I opened a feedback and attached the include data. However, the record feature did NOT record anything useful. On Win 11 Pro, all I did to cause/eliminate the issue was as follows: 
- In Notepad, removed ` "elevate": true,`  from settings.json and saved.
- entered  `wt`  at an Admin command prompt & terminal started.  Closed wt.
- In Notepad, added  `"elevate": true,`  back into settings.json and saved.
- entered  `wt`  at Admin command prompt. Terminal did NOT start, mouse pointer alternating between arrow & arrow+hourglass
- In Notepad, removed  `"elevate": true,`  again and saved. AS SOON AS this settings.json was saved, wt started!

I'm attaching the current settings.json file again. Also, I don't know what you're referring to by "2003 variant". For testing purposes I ran the existing version, 1.14.1963.0.
[termsetjson.zip](https://github.com/microsoft/terminal/files/9294768/termsetjson.zip)

**Reply by DHowett** [reply]
> My account is an admin account.

Oh, this is really interesting. Do you have UAC disabled?

**Reply by cniggeler** [reply]
Good guess! On Win11 Pro, I have UAC set to "Never Notify". On my Win10 Pro machine I I took the extra step of Registry setting to disable it completely (EnableLUA -> 0). Don't know if that is equivalent to "never notify", but I don't believe I changed the Registry in Win11.

EDIT: Checked the Win11 system and confirmed the same Registry setting EnableLUA  is also 0. I don't recall changing it by hand, but I do recall setting the slider in Control Panel to Never Notify.

**Comment by cniggeler** [comment]


**Reply by cniggeler** [reply]
> I am also getting curious display behavior:
[ deleted original comment, contained several JPEGs mostly showing dialog buttons devoid of text or colors ]

I was UNABLE to replicate this issue on a Win11 system with Intel graphics. My current system is Win10, HP laptop, nvidia graphics. I uninstalled and reverted to prior version to if it was an issue there too... it was! So this is a non-issue for this beta. Should I report this elsewhere?

**Reply by zadjii-msft** [reply]
curiously, we've seen these both before

* #13428
* #13382

**Reply by cniggeler** [reply]
Thanks for finding these! I added a comment to #13382 - found it occurs in Windows 10 but cannot replicate in Win11. As for #13428 I guess I'll have to live with it as it seems to be a Win10 issue, but it's minor cuz the text lines up if you enlarge the window size.
